### PR TITLE
Add config scaffolding

### PR DESCRIPTION
**Pull request by dottorblaster**
As above 🤘

### CODE REVIEW

File: lib/mix/tasks/hauer.ex [hunk 1]
```diff
@@ -18,12 +18,15 @@ defmodule Mix.Tasks.Hauer do
...
       "update" ->
+        Mix.shell().info("Updating configuration with #{resource_name}")
         update_config(resource_name)
         :ok
       "resource" ->  
         Mix.shell().info("Creating #{resource_name}")
         new_resource(resource_name)
         :ok
+      _ ->
+        Mix.shell().info("I'm afraid I can't do that, Dave.")
```
Comment (korsmakolnikov) [inline lib/mix/tasks/hauer.ex]: I love you.

File: lib/mix/tasks/hauer.ex [hunk 2]
```diff
@@ -57,7 +60,10 @@ defmodule Mix.Tasks.Hauer do
   """
   def update_config(resource_name) do
     IO.puts(resource_name)
-    # {:ok, pwd} = File.cwd()
-    conf = Hauer.Configuration.read()
+    {:ok, pwd} = File.cwd()
+
+    updated_resources = Hauer.Configuration.read()
```
Comment (korsmakolnikov) [inline lib/mix/tasks/hauer.ex]: Since this is configuration related move in Hauer.configuration

File: lib/hauer/configuration.ex [hunk 3]
```diff
@@ -25,17 +25,16 @@ defmodule Hauer.Configuration do
...
   end
 
-
-  def parse_conf nil do
+  def parse_conf(nil) do
     []
   end
 
-  def parse_conf resources do
+  def parse_conf(resources) do
```
Comment (korsmakolnikov) [inline lib/hauer/configuration.ex]: I wish you move this in a "parser" module. I feel this function has to grow as we introduce new feature in configuration.

Comment (dottorblaster) [reply lib/hauer/configuration.ex]: I think this is not related to this PR. I mean, I only give that a formatter shot :D

File: lib/mix/tasks/hauer.ex (hunk 2 above)
Comment (dottorblaster) [reply lib/mix/tasks/hauer.ex]: Makes sense

**Comment by dottorblaster** [comment]
@korsmakolnikov please check my fixup!

**Pull request merged**